README.md

```diff
@@ -574,7 +574,6 @@
 <details>
 <summary>Strings</summary>
 
```

Comment:
I initially thought about adding a different problem I practised today. But then just did what you suggested.